packager/util/config.go

```diff
@@ -24,13 +24,17 @@ import (
 )
 
 type Config struct {
-	LocalOnly bool
-	Port      int
-	CertFile  string // This must be the full certificate chain.
-	KeyFile   string // Just for the first cert, obviously.
-	OCSPCache string
+	LocalOnly	bool
+	Port		int
+	CertFile	string // This must be the full certificate chain.
+	KeyFile		string // Just for the first cert, obviously.
+	NewCertFile	string // The new full certificate chain replacing the expired one.
+	NewKeyFile	string // For the first cert in NewCertFile.
```

Comment:
I don't think NewKeyFile is necessary for v1. (Correct me if I'm wrong!) It looks like ACME does have a key change protocol (https://tools.ietf.org/html/rfc8555#section-7.3.5) but the usual 90-day renewal won't involve it AFAIK.

Reply:
You are correct, I will remove the field.